src/redux/reducers/cellSets/helpers.js

```diff
@@ -1,10 +1,12 @@
-const createHierarchyFromTree = (data) => data && data.map((rootNode) => {
+const createHierarchyFromTree = (data, order = null) => data && data.map((rootNode) => {
```

Comment:
Why is order null? Is there a case in which we will not want to preserve order?

Reply:
There are other calls to `createHierarchyFromTree` in which we might not want to specify an order. In these cases, `order` is not passed to the function. We can not set a default value, in which case `order` becomes `undefined` in those cases and will still be evaluated the same, in regards to the use in this function.